books/RayTracingInOneWeekend.html

```diff
@@ -502,54 +502,93 @@
 
 Sending Rays Into the Scene
 ----------------------------
-Now we are ready to turn the corner and make a ray tracer. At its core, a ray tracer sends rays
-through pixels and computes the color seen in the direction of those rays. The involved steps are
+Now we are ready to turn the corner and make a ray tracer.
+At its core, a ray tracer sends rays through pixels and computes the color seen in the direction of
+  those rays.
```

Comment:
?

Reply:
Can you elaborate?

Reply:
Why is there `  ` sprinkled about at the beginning of lines?

Reply:
I'm authoring using semantic linefeeds. Wrapped sentences get indented. This makes it easier to diff text changes  without having to track whole-paragraph reflow. Unfortunately, the reflow issue happens once when going from prose style to semantic linefeeds.